support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -188,13 +188,15 @@ function buildReferrerAcquisitionData(acquisitionData: Object): ReferrerAcquisit
     acquisitionData.queryParameters ||
     toAcquisitionQueryParameters(getAllQueryParamsWithExclusions(parameterExclusions));
 
+  const establishedSource = (/\bPPC\b/i.test(campaignCode) || getQueryParameter('glcid')) ? 'PPC' : acquisitionData.source;
```

Comment:
flow isn't happy - `Cannot call `/\bPPC\b/i.test` with `campaignCode` bound to `string` because null or undefined [1] is incompatible with`
How about: `(campaignCode && /\bPPC\b/i.test(campaignCode)) || getQueryParameter('glcid')`

Reply:
🤦